### DISCUSSION TITLE
How to make plain URL in markdown files shows as links instead of text

### DISCUSSION THREAD

**Discussion by damascene** [getpelican/pelican#3180] [Q&A]
I've tried to find a solution and searched for a simple solution to have links like this: `https://example.com` in `md` file showing as an actual link in the html output, however they show only as un-clickable text. 

Any workaround or solution for this?

**Comment by justinmayer** [comment]
My understanding is that what you are observing is the correct, standard behavior for Markdown. To have the URL rendered as a link to the URL itself, wrap it in angle brackets: `<https://example.com>`

**Comment by damascene** [comment]
@justinmayer right, this is the default behavior, however to my knowledge most implementations render it to link. I was looking for a method to enable that.

**Reply by justinmayer** [reply]
Most implementations do a lot of things that are not in the original, canonical Markdown specification. 🤷 

I am not aware of a method to accomplish your stated objective — i.e., self-linking without angle brackets or something like `[https://example.com](https://example.com)`.
